analyzers/src/SonarAnalyzer.Common/Rules/UseUnixEpochBase.cs

```diff
@@ -33,7 +33,8 @@ public abstract class UseUnixEpochBase<TSyntaxKind, TLiteralExpression, TMemberA
     where TLiteralExpression : SyntaxNode
     where TMemberAccessExpression : SyntaxNode
 {
-    private const int EpochYear = 1970;
+    private const long EpochTicks = 0x89F7FF5F7B58000;
+    private const int EpochYear = 1970; // 621_355_968_000_000_000
```

Comment:
Why this comment at the year? That is the number of ticks. If we want to add that as a comment it should go with the ticks. However, I think it is not needed.

Reply:
This comment was about not forgetting to add once again that test case. I failed :D. Thanks for noticing it I'll fix it immediately